<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table_system">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="combobox_method">
       <item>
        <property name="text">
         <string>Gauss method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kramer method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seidel method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Simple iteration method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Upper relaxation method</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(One more method)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushbutton_solve">
       <property name="text">
        <string>Solve with chosen method</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushbutton_solve_all">
       <property name="text">
        <string>Solve with all methods</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_time_measurement">
       <property name="text">
        <string>Enable time measurement</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>470</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progress">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_start"/>
   <addaction name="action_help"/>
   <addaction name="action_about"/>
  </widget>
  <action name="action_start">
   <property name="text">
    <string>Start over</string>
   </property>
   <property name="toolTip">
    <string>Start over</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
